freqtrade/exchange/exchange.py

```diff
@@ -1964,6 +1964,37 @@ def get_funding_fees(self, pair: str, amount: float, open_date: datetime) -> flo
         else:
             return 0.0
 
+    @retrier
+    def get_liquidation_price(self, pair: str):
+        '''
+            Set's the margin mode on the exchange to cross or isolated for a specific pair
+            :param pair: base/quote currency pair (e.g. "ADA/USDT")
+        '''
```

Comment:
Dedent and double quotes, please 
we don't use ''' anywhere for docstrings - and it's something i've never seen used anywhere.